cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,588 @@
+     */
+    const takeSnapshots = (detail: string) => {
+        // Check the status of the seek bar
+        // TODO: check if visible - currently visibility check on a narrow timeline causes an error
+        cy.get(".mx_AudioPlayer_seek input[type='range']").should("exist");
+
+        // Enable group layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+
+        // Click the timestamp to highlight the event tile in case it is not visible
+        cy.get(".mx_EventTile_last[data-layout='group'] .mx_MessageTimestamp").click();
+
+        // Assert that rendering of the player settled and the play button is visible before taking a snapshot
+        checkPlayerVisibility();
+
+        // Take a snapshot on group layout
+        cy.get(".mx_MAudioBody").percySnapshotElement(detail + " on group layout", {
+            percyCSS,
+            widths: snapshotWidthsGroup,
+        });
+
+        cy.log("Took a snapshot of " + detail + " on group layout");
+
+        // Enable bubble layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+
+        // Click the timestamp to highlight the event tile in case it is not visible after changing the layout
+        cy.get(".mx_EventTile_last[data-layout='bubble'] .mx_MessageTimestamp").click();
+
+        checkPlayerVisibility();
+
+        // Take a snapshot on bubble layout
+        cy.get(".mx_MAudioBody").percySnapshotElement(detail + " on bubble layout", {
+            percyCSS,
+            // 243px + 12px + 48px = 303px. See _EventBubbleTile.pcss and _AudioPlayer.pcss for spacing
+            widths: snapshotWidthsBubble,
+        });
+
+        cy.log("Took a snapshot of " + detail + " on bubble layout");
+
+        // Reset the layout
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+    };
+
+    beforeEach(() => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, TEST_USER).then(() =>
+                cy.createRoom({}).then((_roomId) => {
+                    roomId = _roomId;
+                }),
+            );
+        });
+
+        cy.injectAxe();
+    });
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should be correctly rendered on IRC layout", () => {
+        // Please note that takeSnapshots() is not to be used here since it does not take snapshots on IRC layout.
+        // The design of it should be same as on modern layout.
+
+        visitRoom();
+
+        // Upload one second audio file with a long file name
+        uploadFile("cypress/fixtures/1sec-long-name-audio-file.ogg");
+
+        // Scroll to the bottom to make the audio player visible for Percy tests
+        cy.get(".mx_MainSplit .mx_ScrollPanel").scrollTo("bottom");
+
+        cy.get(".mx_RoomView_MessageList").within(() => {
+            checkPlayerFilenameLong();
```

Comment:
The intention of it was to ensure that the audio player renders every information properly, but it indeed seems `checkPlayerFilenameLong` itself is no longer necessary / relevant because Percy checks a visual regression. What would you think?